backend/routes/proposal.js

```diff
@@ -155,7 +155,14 @@ router.post("/update", recaptchaCheck(0.5), checkSigner, function (req, res) {
         { runValidators: true },
         async (err, data) => {
           if (err) return res.status(400).send(err);
-          await updateAirtableEntry(airtableId, update); // update airtable entry
+
+          await updateAirtableEntry(airtableId, update).then((resp) => {}, error => {
```

Comment:
Please revert this part, it was fixed by https://github.com/oceanprotocol/oceandao-proposal-portal/pull/122